src/charm.py

```diff
@@ -65,17 +65,9 @@ def _on_install(self, _: ops.InstallEvent) -> None:
                     "Secret with label %s not found. Creating one.", WAZUH_CLUSTER_KEY_SECRET_LABEL
                 )
                 self.app.add_secret(
-                    {"value": secrets.token_hex(8)}, label=WAZUH_CLUSTER_KEY_SECRET_LABEL
+                    {"value": secrets.token_hex(16)}, label=WAZUH_CLUSTER_KEY_SECRET_LABEL
```

Comment:
Looking at the amount of places where this change was requires, shouldn't it use a constant ?

Reply:
One place in the sources, several in the tests. If I share a constant between sources and tests, it's basically like testing nothing, isn't it?